<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleGui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton1">
      <property name="text">
       <string>Push me</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLCDNumber" name="lcdNumber1"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton2">
      <property name="text">
       <string>No me!</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLCDNumber" name="lcdNumber2"/>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton3">
      <property name="text">
       <string>Meee!</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLCDNumber" name="lcdNumber3"/>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="LivePlotWidget" name="plot_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="save_data_button">
      <property name="text">
       <string>Save Data</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="filename_edit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="connect_button">
      <property name="text">
       <string>Connect to server</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LivePlotWidget</class>
   <extends>QWidget</extends>
   <header>pglive.sources.live_plot_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
